man/systemd-analyze.xml

```diff
@@ -1713,6 +1713,14 @@ LEGEND: M → sys_vendor (LENOVO) ┄ F → product_family (ThinkPad X1 Carbon G
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--drm-device=<replaceable>DRM</replaceable></option></term>
+
+        <listitem><para>When provided with the <command>chid</command> command, use this sysfs path to a DRM device to fetch EDID from. Example: <filename>/sys/class/drm/card1-HDMI-A-1</filename></para>
```

Comment:
we have this rule in systemd that in docs and log messages if we know that a path refers to a dir (or a symlink to a dir) then we should suffix the path with a /. This path here qualifies, hence please add a trailing slash